sympy/stats/crv_types.py

```diff
@@ -149,12 +150,6 @@ def __new__(cls, pdf, set=Interval(-oo, oo)):
     def set(self):
         return self.args[1]
 
-    @staticmethod
-    def check(pdf, set):
-        x = Dummy('x')
-        val = integrate(pdf(x), (x, set))
-        _value_check(val == S.One, "The pdf on the given set is incorrect.")
-
```

Comment:
I have removed these checks as they raise an error in some cases even if the pdf is integrated to 1.